src/app-layout/index.tsx

```diff
@@ -115,6 +115,9 @@ const OldAppLayout = React.forwardRef(
       onSplitPanelToggle,
       onNavigationChange,
       onToolsChange,
+      publicDrawers,
```

Comment:
I was getting a build error when I just called this `drawers`, so I updated it to `publicDrawers` for the time being. For the final launch, we'll have to change this to `drawers`, but I wasn't sure how to avoid the naming conflict with the beta drawers without breaking anything. Would love any ideas.